### PR TITLE
Update to RockyLinux 8 build container + new PRs (AB#22732)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: tobuild.json [hunk 1]
```diff
@@ -27,11 +27,10 @@
   "configuration-modules-core": {
     "branch": "master",
     "prs": [
-      1438,
-      1452,
-      1510
+      1695,
+      1697
```
Comment (StephaneGerardVUB) [inline tobuild.json]: this one has been merged into master -> can be removed

File: tobuild.json [hunk 2]
```diff
@@ -27,11 +27,10 @@
...
     ],
-    "toversion": "21.12.1"
+    "toversion": "23.06.0"
```
Comment (StephaneGerardVUB) [inline tobuild.json:32]: Not sure about the format. Shouldn't it be '23.6.0'?

File: tobuild.json (hunk 1 above)
Comment (wpoely86) [reply tobuild.json]: done

**Review by StephaneGerardVUB — APPROVED** [review]
lgtm

**Pull request merged**